warehouse/age-off/src/main/java/datawave/ingest/util/cache/watch/FileRuleWatcher.java

```diff
@@ -89,37 +88,13 @@ public FileRuleWatcher(Path filePath, long configuredDiff, IteratorEnvironment i
     @Override
     protected Collection<FilterRule> loadContents(InputStream in) throws IOException {
         try {
-            AgeOffRuleLoader ruleLoader = new AgeOffRuleLoader(new FileWatcherDependencyProvider());
+            AgeOffRuleLoader ruleLoader = new AgeOffRuleLoader(new FileLoaderDependencyProvider(fs, filePath, iterEnv));
             return ruleLoader.load(in);
         } catch (Exception ex) {
```

Comment:
likewise, update to be more specific

Reply:
Removed general exception catch - `IOException` remained on the method as a checked exception.